assets/files.wxs

```diff
@@ -3105,11 +3105,8 @@
       <Component Id="cmpC0FFB3F4FB30438082D2DC0F4E4BF12D">
         <File Id="fil8FEEEB3B27DF40148EB85CE6AE1A420B" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.DiaSymReader.Native.$(var.FileArchitecture).dll" />
       </Component>
-      <Component Id="cmp5485813EC16244898CB9A944AC2BC5E9">
-        <File Id="fil6EA784763CF94299BC6E75DF068F05A3" KeyPath="yes" Source="$(var.ProductSourcePath)\Microsoft.PowerShell.Pager.dll" />
-      </Component>
-      <Component Id="cmp9AFE828DDCCC4A0E83C4D7476B353023">
-        <File Id="filBE5924015B7147E5BE1D6CDEB8AFB6C3" KeyPath="yes" Source="$(var.ProductSourcePath)\mscordaccore_$(var.FileArchitecture)_$(var.FileArchitecture)_5.0.20.40711.dll" />
```

Comment:
The `5.0.20.40711.dll` was an update in the .NET-preview8 PR (see https://github.com/PowerShell/PowerShell/commit/6a37eb9313077f726c4bca7902466a1f5a6a3f44#diff-a4b82ba0eee41f8e27cca51303335950). I think this is the one that should be kept.

Reply:
CI will regenerate